Gems/AtomLyIntegration/EMotionFXAtom/Code/Tools/EMStudio/AnimViewportWidget.cpp

```diff
@@ -209,16 +218,22 @@ namespace EMStudio
             break;
         case RenderOptions::CameraViewMode::DEFAULT:
             // The default view mode is looking from the top left of the character.
-            cameraPosition.Set(
-                targetPosition.GetX() - CameraDistance, targetPosition.GetY() + CameraDistance, targetPosition.GetZ() + CameraDistance);
+            const AZ::Vector3 displacementVector = AZ::Vector3::CreateOne().GetNormalized() * CameraDistance;
+            cameraPosition = targetPosition + AZ::Vector3(-displacementVector.GetX(), displacementVector.GetY(), displacementVector.GetZ());
```

Comment:
[nit] Maybe less operations and simpler to invert the X and not calling all the getters and create a new vector?

````
AZ::Vector3 displacementVector = AZ::Vector3::CreateOne().GetNormalized() * CameraDistance;
displacementVector.SetX(-displacementVector.GetX());
cameraPosition = targetPosition + displacementVector;
````

Reply:
Yeah agreed I was trying to think of a nice way to write this (as before it was technically wrong), I just had another look with fresh eyes and after seeing your suggestion and think this might be even nicer

```
const AZ::Vector3 displacementVector = AZ::Vector3(-1.0f, 1.0f, 1.0f).GetNormalized() * CameraDistance;
cameraPosition = targetPosition + displacementVector;
```

As then we get to keep the `const` 😁

Reply:
Even better.